mempool/src/error/ban_score.rs

```diff
@@ -298,3 +297,29 @@ impl MempoolBanScore for pos_accounting::Error {
         }
     }
 }
+
+impl MempoolBanScore for accounting::Error {
+    fn mempool_ban_score(&self) -> u32 {
+        match self {
+            // These should not happen with valid transactions
+            accounting::Error::ArithmeticErrorToUnsignedFailed => 100,
+            accounting::Error::ArithmeticErrorToSignedFailed => 100,
+            accounting::Error::ArithmeticErrorSumToUnsignedFailed => 100,
+            accounting::Error::ArithmeticErrorSumToSignedFailed => 100,
+
+            // These depend on the current state which may be out of sync
+            accounting::Error::ArithmeticErrorDeltaAdditionFailed => 0,
```

Comment:
I'd make it 100 as well so no one tries to overflow

Reply:
done